lib/model/QubitSlug.php

```diff
@@ -85,14 +95,14 @@ public static function slugify($slug, $creationType = null)
                     $slug = $result;
                 }
 
-                // Remove apostrophes
-                $slug = preg_replace('/\'/', '', $slug);
                 $slug = strtolower($slug);
                 // Allow only digits, letters, and dashes.  Replace sequences of other
                 // characters with dash
-                $slug = preg_replace('/[^0-9a-z]+/', '-', $slug);
+                $slug = preg_replace('/[^'.self::SLUG_RESTRICTIVE_CHARS.']+/', '-', $slug);
         }
 
+        // Replace repeating dashes in slug with single dash.
+        $slug = preg_replace('/-+/', '-', $slug);
         $slug = "-{$slug}-";
```

Comment:
I can't see any need for this, as the hyphens added are immediately trimmed on line 109. 😕  I think this line should be removed.